packages/teuchos/core/src/Teuchos_Array.hpp

```diff
@@ -407,9 +407,19 @@ class Array
   /** \brief Return a raw pointer to beginning of array or NULL if unsized. */
   inline T* getRawPtr();
 
+  /// \brief Return a raw pointer to beginning of array.
```

Comment:
@mhoemmen, does Doxygen render this comment correctly using `///`?

Reply:
@bartlettroscoe I've had success doing that with Tpetra.  Thanks for asking, though!